index.html

```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
     </a>
     <img class="mosaic-img" src="./Media/image_geometry_2.svg" alt=" ">
   </section>
+  <section class="works">
+    <h2>Projects</h2>
+    <img class="works-img" src="./Media/work-img-1.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
+    <img class="works-img" src="./Media/work-img-2.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-3.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-4.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-5.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

Reply:
Satisfied.